### PR TITLE
VSTS-10935 - Background Jobs

### PR DESCRIPTION
**Pull request by jessedoyle**
:elephant:

* Enable systemd as a process monitor for our Shoryuken background
  worker processes. We need to ensure that systemd starts the
  the workers when the server is rebooted or if a process crashes.
* Don't start the Shoryuken process using the `daemon` mode so that
  systemd is able to properly daemonize the process itself. See
  http://0pointer.de/public/systemd-man/daemon.html#New-Style%20Daemons.
* Configure systemd to start the process with the proper environment
  variables by specifying the EnvironmentFile directive.

SEE: https://amaabca.visualstudio.com/driver_education_backlog/_workitems/edit/10935

### CODE REVIEW

**Comment by rubene** [comment]
:book:

File: meta/main.yml [hunk 1]
```diff
@@ -1,57 +1,20 @@
...
-    # List tags for your role here, one per line. A tag is a keyword that describes
-    # and categorizes the role. Users find roles by searching for tags. Be sure to
-    # remove the '[]' above, if you add tags to this list.
-    #
-    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
-    #       Maximum 20 tags per role.
+  galaxy_tags:
+    - systemd
+    - shoryuken
+    - ubutnu
```
Comment (rubene) [inline meta/main.yml]: typo on ubuntu

File: meta/main.yml [hunk 2]
```diff
@@ -1,57 +1,20 @@
...
 
 dependencies: []
-  # List your role dependencies here, one per line. Be sure to remove the '[]' above,
-  # if you add dependencies to this list.
```
Comment (rubene) [inline meta/main.yml]: new line

**Review by rubene — CHANGES_REQUESTED** [review]

**Comment by jessedoyle** [comment]
:wave:

**Comment by rubene** [comment]
🐘 🐘 🐘

**Review by rubene — APPROVED** [review]

**Review by rubene — APPROVED** [review]

**Pull request merged**